calour/io.py

```diff
@@ -345,6 +345,7 @@ def _read_metadata(ids, f, kwargs):
 
 @ds.get_sectionsf('io.read')
 def read(data_file, sample_metadata_file=None, feature_metadata_file=None,
+         call_history_file=None,
```

Comment:
I think important since otherwise you get partial/missing history
We can do that if call_history is True (which we set as default instead of None), can try to automatically load the call history based on the data_file filename (as created in the save() ) if it exists.
I am afraid if we do it a separate function, people will be lazy/not know about it and then they lose the history once they load a saved file and continue working

What to you think?